CHANGELOG.md

```diff
@@ -5,10 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [3.XX.XX] - 2023-XX-XX
+## [3.10.2] - 2023-05-16
 
 ### Fixed
 - Added PXHD from risk response to the async activities
+- Fixed custom cookie header overrides the regular Cookies header
```

Comment:
Suggestion:
### Added
Custom cookie header is processed in addition to (not instead of) default cookie header
Custom cookie header default value has been set to x-px-cookies
- Added PXHD from risk response to the async activities